test/specs/utils/query.js

```diff
@@ -27,29 +27,57 @@ describe('Query class', () => {
-						Then('the lisk instance should send a request to the blocks/get API endpoint with the block ID', then.theLiskAPIInstanceShouldSendARequestToTheBlocksGetAPIEndpointWithTheBlockID);
+					When('request is successful', when.sendRequestSuccesses, () => {
+						When('the query instance gets a block using the ID', when.theQueryInstanceGetsABlockUsingTheID, () => {
+							Then('the lisk instance should send a request to the blocks/get API endpoint with the block ID', then.theLiskAPIInstanceShouldSendARequestToTheBlocksGetAPIEndpointWithTheBlockID);
+						});
+					});
+					When('request fails', when.sendRequestFails, () => {
+						When('the query instance gets a block using the ID', when.theQueryInstanceGetsABlockUsingTheID, () => {
+							Then('the query should be rejected with an error message', then.itShouldRejectWithTheErrorMessage);
+						});
 					});
 				});
 			});
 			describe('#getAccount', () => {
 				Given('an address "13782017140058682841L"', given.anAddress, () => {
-					When('the query instance gets an account using the address', when.theQueryInstanceGetsAnAccountUsingTheAddress, () => {
-						Then('the lisk instance should send a request to the accounts API endpoint with the address', then.theLiskAPIInstanceShouldSendARequestToTheAccountsAPIEndpointWithTheAddress);
+					When('request is successful', when.sendRequestSuccesses, () => {
+						When('the query instance gets an account using the address', when.theQueryInstanceGetsAnAccountUsingTheAddress, () => {
+							Then('the lisk instance should send a request to the accounts API endpoint with the address', then.theLiskAPIInstanceShouldSendARequestToTheAccountsAPIEndpointWithTheAddress);
+						});
+					});
+					When('request fails', when.sendRequestFails, () => {
+						When('the query instance gets an account using the address', when.theQueryInstanceGetsAnAccountUsingTheAddress, () => {
+							Then('the query should be rejected with an error message', then.itShouldRejectWithTheErrorMessage);
+						});
 					});
 				});
 			});
 			describe('#getTransaction', () => {
 				Given('a transaction ID "16388447461355055139"', given.aTransactionID, () => {
-					When('the query instance gets a transaction using the ID', when.theQueryInstanceGetsATransactionUsingTheID, () => {
-						Then('the lisk instance should send a request to the transactions/get API endpoint with the transaction ID', then.theLiskAPIInstanceShouldSendARequestToTheTransactionsGetAPIEndpointWithTheTransactionID);
+					When('request is successful', when.sendRequestSuccesses, () => {
+						When('the query instance gets a transaction using the ID', when.theQueryInstanceGetsATransactionUsingTheID, () => {
+							Then('the lisk instance should send a request to the transactions/get API endpoint with the transaction ID', then.theLiskAPIInstanceShouldSendARequestToTheTransactionsGetAPIEndpointWithTheTransactionID);
```

Comment:
`transactions/get` is technical language, not English, so it should be in quotes. (Also applies to other endpoints.)